wsd/COOLWSD.cpp

```diff
@@ -4030,10 +4030,6 @@ int COOLWSD::innerMain()
 
     SigUtil::addActivity("finished with status " + std::to_string(returnValue));
 
-    // At least on centos7, Poco deadlocks while
-    // cleaning up its SSL context singleton.
-    Util::forcedExit(returnValue);
```

Comment:
ok, innerMain is only called from main, which always calls cleanup, so move this force exit into cleanup instead and release everything unproblematic and then just skip the ssl deinitialization.

Sounds good, but I guess that means this cleanup path was mostly unused before this change then.

Reply:
Yes, was unused for a long time, which is why when I enabled it we saw the double-free errors in jenkins (it had bit-rot). As the comment says, at one point we noticed on CentOS Poco SSL uninitialization was getting deadlocked, so we forcefully exited.

But now I needed to chase leaks, and this is in the way.